crates/turbopack-tests/tests/snapshot.rs

```diff
@@ -39,6 +40,7 @@ use turbopack_env::ProcessEnvAssetVc;
 
 fn register() {
     turbopack::register();
+    next_core::register();
```

Comment:
@jridgewell To address https://github.com/vercel/turbo/pull/2742/files#r1036077301, I made `add_next_font_transform` public and use it here. This achieves the same result as the ModuleOptionsContext field but unfortunately leaks these internals into the snapshot test runner. What do you think?

Reply:
Feel free to disregard -- I'm going to see how this looks as an integration test in next-dev instead.